app/templates/about.html

```diff
@@ -6,10 +6,10 @@
 </head>
     <body>
         <ul>
-          <li><a href="about.html">About</a></li>
-          <li><a href="search.html">Search</a></li>
-          <li><a href="upload_recipe_form.html">Submit</a></li>
-          <li><a href="sign_in.html">Sign In</a></li>
+          <li><a href="about">About</a></li>
+          <li><a href="search">Search</a></li>
+          <li><a href="upload_recipe">Submit</a></li>
+          <!-- <li><a href="sign_in.html">Sign In</a></li> -->
         </ul>
         <h1>About BlindOven</h1>
```

Comment:
We should replace 'BlindOven' with 'Sightless Chef', but that doesn't have to be for this PR